README.md

```diff
@@ -1,9 +1,28 @@
-# LAP 1 Code Challenge: Gogol
+# LAP 1 Code Challenge: Google
 
 ## By Sarushan and Rubhan
 
 ## Assignment Description
 
+Trying to replicate the google search
+
 ## Installation
 
-## Problems and Challenges.
+- Express
+- Cors
```

Comment:
Your installation can be a lot more descriptive. Exactly where should I install these packages? Do I have to cd into anything before I install these? Also write down the command I should drop into the terminal